packages/gatsby-cli/src/reporter/index.js

```diff
@@ -59,6 +59,9 @@ const reporter: Reporter = {
     let details = {}
     // Many paths to retain backcompat :scream:
     if (arguments.length === 2) {
+      if (Array.isArray(error)) {
```

Comment:
I don't see an immediate need for this in your usage of error reporting but a good change nonetheless 👍

Reply:
Ah yeah, the need for this in case of webpack error went away, but there might be other similar cases that are not handled that we just don't know about